website/static/js/fangorn.js

```diff
@@ -1061,12 +1654,35 @@ tbOptions = {
         $(document).on('click', '.fangorn-dismiss', function() {
```

Comment:
I know this isn't a part of this PR, but is this click handler necessary? I don't see the `fangorn-dismiss` class applied anywhere.